jane/doc/extensions/uniqueness/intro.md

```diff
@@ -0,0 +1,121 @@
+# Introduction to Uniqueness
+
+See also the full feature [reference](reference.md) and [common pitfalls](pitfalls.md).
+In this document, we use the new [syntax for modes](../modes/syntax.md),
+which we expect to be stable soonish.
+
+The `unique` mode designates values that have only a single reference pointing
+to them. For example, an externally allocated data structure might support a
```

Comment:
This isn't directly an example of a value with a single reference - something like 'uniqueness is useful in situations like freeing an externally allocated data structure...'

Reply:
Thanks, it now reads:

> The `unique` mode designates values that have only a single reference pointing
to them. If an operation takes a `unique` argument, it will consume the only
reference to the value. For example, an externally allocated data structure
might support a `free` operation: